src/rrr_riv_tot_scl_prm.py

```diff
@@ -88,21 +89,25 @@
 #kfac file
 #-------------------------------------------------------------------------------
 ZV_kfac=[]
+huc_8_list_k=[]
 with open(rrr_kfc_file,'rb') as csvfile:
      csvreader=csv.reader(csvfile)
      for row in csvreader:
-          ZV_kfac.append(float(row[0]))
+          huc_8_list_k.append(row[0])
```

Comment:
is this the only place where rrr_kfc_file gets used? just want to make sure rapid doesn't have issues with adding another entry to the row

Reply:
Yes it is, at least in our pipeline.